### ISSUE TITLE
Cannot set `dipy` as a dependency

### ISSUE THREAD

**Issue by casperdcl** [dipy/dipy#2747]
## Way to reproduce
```python
$ ls
setup.py
$ cat setup.py
from setuptools import setup
setup(version="0.0.0", install_requires=['dipy'])
$ python setup.py install
...
pkg_resources.extern.packaging.requirements.InvalidRequirement: Expected closing RIGHT_PARENTHESIS
    fury (>=0.8.0scikit-learn) ; extra == 'all'
         ~~~~~~~~^
```

Completely clean conda environment, tested on Py3.11 & Py3.9.

Putting anything else in `install_requires` works without issue (e.g. `install_requires=['numpy', 'fury']`) so there's no problem with my system.

Perhaps there's some wrong line endings in `dipy`'s requirements/dependency spec somewhere? Or perhaps https://github.com/dipy/dipy/blob/770448108d95f705b64a8ded074b6146695b409e/dipy/info.py#L128-L129 isn't parsed properly?

<details><summary>system/version details</summary>

- Linux-5.15.90.1-microsoft-standard-WSL2-x86_64-with-glibc2.31
- Python 3.11.0 | packaged by conda-forge | (main, Jan 14 2023, 12:27:40) [GCC 11.3.0]
- dipy 1.6.0
- setuptools 67.6.0
- NumPy 1.24.2
- SciPy 1.10.1
- Nibabel 5.0.1
- H5py 3.8.0
- ModuleNotFoundError: No module named 'cvxpy'
- fury 0.8.0

</details>

**Comment by skoudoro** [comment]
Hi @casperdcl,

Which version are you using ?  it has been fixed on the main branch recently via #2727.  But the problem still remains on the last release. we plan a new release on April

**Comment by casperdcl** [comment]
Ah that's why my code search didn't find anything. It's already fixed; just not released yet.

**Comment by skoudoro** [comment]
closing, fixed by the last release 1.7.0
